File: src/main/java/tooko/main/extras/LICENCE.java
package tooko.main.extras;

import tooko.main.*;
import tooko.td.TdApi.*;
import tooko.td.client.*;

public class LICENCE extends TdHandler {

    @Override
    public void onLoad() {

        initFunction("license");

    }

    @Override
    public void onFunction(User user, long chatId, Message message, String function, String param, String[] params, String[] originParams) {

        send(Fn.sendText(chatId, Fn.parseHtml(Lang.get(user).LICENSE)));

    }

}

File: src/main/java/tooko/core/funs/LICENCE.kt
/*
 * Copyright (c) 2019 - 2020 KazamaWataru
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *  http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package tooko.core.funs

import td.TdApi
import tooko.core.env.Lang
import tooko.core.client.*
import tooko.core.utils.*

class LICENCE : TdBotHandler() {

    override fun onLoad() = initFunction("license")

    override suspend fun onFunction(userId: Int, chatId: Long, message: TdApi.Message, function: String, param: String, params: Array<String>, originParams: Array<String>) {

        sudo makeHtml Lang.get(userId).LICENSE sendTo chatId

    }

}